workbench/database/factories/UserEntityFactory.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Workbench\Database\Factories;
+
+use Faker\Generator;
+use LaravelDoctrine\ORM\Testing\Factory;
+use Workbench\App\Entities\User;
+
+/** @var Factory $factory */
+$factory->define(User::class, static function (Generator $faker, array $attributes = []) {
+    return [
+        'name' => $faker->name(),
+        'email' => $faker->safeEmail,
+        'password' => 'password',
```

Comment:
If password is always set to 'password' how can the unit test pass looking for 'abc'?

Reply:
these are just default values which can be overwritten by passing in an array to make() which is what I do in the test file on line 39:
`$user = entity(User::class)->make(['password' => 'abc']);`

these values are merged with the default values when hydrating the object, see https://github.com/laravel-doctrine/orm/blob/04915c1ecd9e888e166c1389d07605527a45e991/src/Testing/FactoryBuilder.php#L230